packages/lib/models/Setting.ts

```diff
@@ -1283,6 +1283,13 @@ class Setting extends BaseModel {
 				isGlobal: true,
 			},
 
+			hideMenuBar: {
+				value: false,
+				type: SettingItemType.Bool,
+				public: false,
+				appTypes: [AppType.Desktop],
+			},
```

Comment:
Sorry would you mind naming it "showMenuBar" and setting it to `true` by default? This is the pattern we try to follow to avoid double-negations.

Reply:
Agreed, I'll change it.

Reply:
Changed in the latest commit.